README.md

```diff
@@ -138,7 +123,118 @@ This temporary directory is then added to the `source_paths` of the Rails
 generator system, allowing all of its ERb templates and files to be referenced
 when the application template script is evaluated.
 
-Rails generators are very lightly documented; what you’ll find is that most of
-the heavy lifting is done by [Thor](http://whatisthor.com/). The most common methods used by this
-template are Thor’s `copy_file`, `template`, and `gsub_file`. You can dig into
-the well-organized and well-documented [Thor source code](https://github.com/erikhuda/thor) to learn more.
+Rails generators are very lightly documented; what you’ll find is that most of the heavy lifting is done by  [Thor](http://whatisthor.com/). Thor is a tool that allows you to easily perform command line utilities.  The most common methods used by this template are Thor’s `copy_file`, `template`, and `gsub_file`. You can dig into the well-organized and well-documented [Thor source code](https://github.com/erikhuda/thor) to learn more.
+If any file finishes with `.tt`, Thor considers it to be a template and places it in the destination without the extension `.tt`. 
+
+## Tooling choices and configuration
+  
+#### Testing: [webdrivers](https://github.com/titusfortner/webdrivers)
+auto-installs headless Chrome
+
+#### ENV managment: [dotenv](https://github.com/bkeepers/dotenv)  
+in place of the Rails `secrets.yml`
+  
+#### Application web server: [puma](https://github.com/puma/puma)
+  
+#### Security Auditing: [brakeman](https://github.com/presidentbeef/brakeman) and [bundler-audit](https://github.com/rubysec/bundler-audit)
```

Comment:
We also use [`yarn audit`](https://classic.yarnpkg.com/en/docs/cli/audit/)/[`npm audit`](https://docs.npmjs.com/cli/audit) for auditing js dependencies.

Reply:
added in a mention of yarn audit
https://github.com/ackama/rails-template/pull/157/commits/d2bc43d5833fb98ea21a7c3ad9631827a3a1974c